src/controller/deletecontent.js

```diff
@@ -99,8 +123,23 @@ function shouldAutoparagraph( doc, position ) {
 	return !isTextAllowed && isParagraphAllowed;
 }
 
-function checkCanBeMerged( element ) {
-	const schema = element.document.schema;
+function checkCanBeMerged( left, right ) {
+	const schema = left.document.schema;
```

Comment:
I haven't even thought about it. So, I'd need to iterate over all elements between those elements. That would be nice. I'll see how the code looks with it.